--- LucenePlusPlus-rel_3.0.8/src/config/core/liblucene++.pc.in~	2020-12-29 13:10:43.000000000 +0100
+++ LucenePlusPlus-rel_3.0.8/src/config/core/liblucene++.pc.in	2023-09-29 09:03:57.213964577 +0200
@@ -1,12 +1,11 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}/bin
-libdir=@LIB_DESTINATION@
+libdir=@CMAKE_INSTALL_LIBDIR@
 includedir=${prefix}/include/lucene++
 lib=lucene++
 
 Name: liblucene++
 Description: Lucene++ - a C++ search engine, ported from the popular Apache Lucene
 Version: @lucene++_VERSION@
-Libs: -L@LIB_DESTINATION@ -l${lib}
+Libs: -L${libdir} -l${lib}
 Cflags: -I${includedir}
-
--- LucenePlusPlus-rel_3.0.8/src/config/contrib/liblucene++-contrib.pc.in~	2020-12-29 13:10:43.000000000 +0100
+++ LucenePlusPlus-rel_3.0.8/src/config/contrib/liblucene++-contrib.pc.in	2023-09-29 09:04:44.724865773 +0200
@@ -1,13 +1,12 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}/bin
-libdir=@LIB_DESTINATION@
+libdir=@CMAKE_INSTALL_LIBDIR@
 includedir=${prefix}/include/lucene++
 lib=lucene++-contrib
 
 Name: liblucene++-contrib
 Description: Contributions for Lucene++ - a C++ search engine, ported from the popular Apache Lucene
 Version: @lucene++_VERSION@
-Libs: -L@LIB_DESTINATION@ -l${lib}
+Libs: -L${libdir} -l${lib}
 Cflags: -I${includedir}
 Requires: liblucene++ = @lucene++_VERSION@
-
