--- LucenePlusPlus-rel_3.0.8/CMakeLists.txt.orig	2020-12-29 13:10:43.000000000 +0100
+++ LucenePlusPlus-rel_3.0.8/CMakeLists.txt	2023-09-28 23:39:19.228743152 +0200
@@ -46,10 +46,6 @@
 # build docs
 include(Lucene++Docs)
 
-# Enable C++11
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-
 ####################################
 # platform specific options
 ####################################
